news/4.33/platform.html

```diff
@@ -141,6 +141,34 @@ <h2>Preferences </h2>
     <h2>Themes and Styling </h2>
     </td>
   </tr>
+  <tr id="new-light-theme"> <!-- https://github.com/eclipse-platform/eclipse.platform.ui/issues/2114 -->
+      <td class="title">Preview Version of Improved Light Theme</td>
+      <td class="content">
+        <p>
+        The Eclipse default light theme has not been updated in years. It is not state 
+        of the art anymore. All major IDEs have shifted to a different light theme style.
+        When comparing the different IDEs it is easy to pick out the design decisions 
+        they all share: 
+        </p>
+        <ol>
+          <li>Lightweight view tab design</li>
+          <li>Flat look - No use of 3D gradients</li>
+          <li>Views use a darker background color.</li>
+        </ol>
+        <p>
+        We have been working hard on an improved version of the light theme that follows these 
+        design decisions. These improvement are not yet finished but are available as a preview 
+        version. 
+        </p>
+        <p>
+        You can try it out by selecting <em>Light (Preview)</em> on the <b>General > 
+        Appearance</b> preference page. We are keen on your feedback. Please 
+        <a href="https://github.com/eclipse-platform/eclipse.platform.ui/issues/2114">give your 
+        feedback</a> and we will look into it.
```

Comment:
`See issue #2114 for more details and the motivation behind these changes and to give your feedback.`